workspaces/bulk-import/plugins/bulk-import-backend/src/schema/openapi.yaml

```diff
@@ -580,6 +594,28 @@ components:
           $ref: '#/components/schemas/ApprovalTool'
         repository:
           $ref: '#/components/schemas/Repository'
+        gitlab:
+          type: object
+          description: GitLab details. Applicable if approvalTool is gitlab.
+          properties:
+            pullRequest:
```

Comment:
Up to you, but note that in GitLab a pull request is called a 'merge request,' so maybe `mergeRequest` is more appropriate.

Reply:
I think it is fine so long as it isn't customer facing (don't want to cause potential confusion where possible).